React/Views/RCTBorderDrawing.m

```diff
@@ -498,6 +497,48 @@ static CGContextRef RCTUIGraphicsBeginImageContext(CGSize size, CGColorRef backg
   return image;
 }
 
+static id RCTGetBorderImageKey(RCTBorderStyle borderStyle,
+                               CGSize viewSize,
+                               RCTCornerRadii cornerRadii,
+                               UIEdgeInsets borderInsets,
+                               RCTBorderColors borderColors,
+                               CGColorRef backgroundColor,
+                               BOOL drawToEdge)
+{
+  // Important that this struct is aligned on all architectures, else
+  // the compiler can insert garbage into the padding and corrupt our hash.
```

Comment:
Yep we can, I initially chose not to for the sake of visual cleanliness but thanks for the feedback, safety is king. Ill drop it in later today.